### PR TITLE
OE-200 create reusable workflow for publishing documents to confluence

### PR DESCRIPTION
**Pull request by tlotempus**

### CODE REVIEW

File: .github/workflows/README.md [hunk 1]
```diff
@@ -85,4 +85,38 @@ jobs:
...
+    types:
+      - closed
+
+jobs:
+  publish_to_confluence:
+    if: github.event.pull_request.merged == true
+    uses: sportsball-ai/actions-hub/.github/workflows/publish-to-confluence.yml@v2
+    secrets: inherit
+    with:
+      comment: "this is version 1" # add an optional comment to the published page
```
Comment (raven-txm) [inline .github/workflows/README.md]: Can we make this comment not be about version? People are liable to try this without thinking.

**Review by raven-txm — APPROVED** [review]
Approved with a NP

File: .github/workflows/README.md [hunk 2]
```diff
@@ -85,4 +85,38 @@ jobs:
...
+      exclude: ".*temp.md" # List of exclude file patterns (regex) that will be applied on markdown file paths.
+      modified_since: 10 # Only upload files that have modifed in the past n minutes.
```
Comment (jweisberg-txm) [inline .github/workflows/README.md:116]: I'm not quite sure I understand the usefulness of this option.

File: .github/workflows/publish-to-confluence.yml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        CONFLUENCE_USERNAME: opex@tempus-ex.com
+        CONFLUENCE_PASSWORD: ${{ secrets.MARKDOWN_CONFLUENCE }}
+        CONFLUENCE_ENDPOINT: https://epf.atlassian.net/wiki
+
+    permissions: write-all
+
+    steps:
+      - name: download and install markdown2confluence
+        run: |
+          curl -LO https://github.com/justmiles/go-markdown2confluence/releases/download/v3.4.6/go-markdown2confluence_3.4.6_linux_x86_64.tar.gz
```
Comment (jweisberg-txm) [inline .github/workflows/publish-to-confluence.yml]: There's a good chance that we want to mirror this and ensure availability.

**Review by jweisberg-txm — APPROVED** [review]
Great work here, just a question about one of the options and a suggestion that we mirror the plugin you're using for the markdown conversion.  Thank you!

File: .github/workflows/README.md (hunk 2 above)
Comment (beiriannydd) [reply .github/workflows/README.md:116]: modified since last update time on confluence perhaps?

Comment (tlotempus) [reply .github/workflows/README.md:116]: docs say "This is particurlarly useful for cron jobs/recurring one-way syncs."  I think the idea is that it compares the update timestamp on the file and only uploads the file if the timestamp is within the specified window of minutes.  I think the idea is that in a release situation, you can tailor the pushing of documents to only those that were just updated as part of the release.

Comment (jweisberg-txm) [reply .github/workflows/README.md:116]: That makes sense, thank you!

File: .github/workflows/publish-to-confluence.yml (hunk 3 above)
Comment (tlotempus) [reply .github/workflows/publish-to-confluence.yml]: fair point.  Let me look into getting this into nexus.

**Pull request merged**